### ISSUE TITLE
Questions about some definitions in "traj_data.json" 

### ISSUE THREAD

**Issue by ruoxinx** [askforalfred/alfred#79]
Thank you for providing this great dataset!

I noticed that the "ObjectName" in the traj_data.json is attached with a string of characters. For example,

> "object_poses": [
>             {
>                 "objectName": "**Mug_2a940808**",
>                 "position": {
>                     "x": 1.83983088,
>                     "y": 0.793715239,
>                     "z": -1.153574
>                 },
>                 "rotation": {
>                     "x": 0.0,
>                     "y": 0.0,
>                     "z": 0.0
>                 }
>             },
>             {
>                 "objectName": "**TissueBox_00aeed40**",
>                 "position": {
>                     "x": 1.98229468,
>                     "y": 0.350982755,
>                     "z": -1.21802711
>                 },
>                 "rotation": {
>                     "x": 0.0,
>                     "y": 0.0,
>                     "z": 0.0
>                 }
>             },

I am not sure of the meanings of these strings of characters. I try to exchange the same type of object (e.g., mug) with different objectNames (e.g., Mug_2a940808 and Mug_291e6e8a) between different trajectories, but the simulator failed. It seems that the objectName is related to the scene plans. 

Could you please provide some references about the definitions of the objectName? Thank you so much!

**Comment by askforalfred** [comment]
Your intuition is correct. These are ids associated with the object mesh,
which differs scene to scene (e.g., slightly differently shaped and colored
coffee mugs). Object meshes are tied to scenes as far as I remember in THOR
2.1.0.

**Comment by ruoxinx** [comment]
Thank you for your prompt response. I guess it is hard to change the shape or color of objects by simply modifying these ids in "traj_data.json" since they are fixed with scenes right now.

**Comment by MohitShridhar** [comment]
That's right, you can't modify object textures unless you edit the scenes in [Unity](https://github.com/allenai/ai2thor) and build your own THOR binary.
